django/contrib/admin/templatetags/admin_list.py

```diff
@@ -128,6 +128,10 @@ def result_headers(cl):
                 continue
 
             admin_order_field = getattr(attr, "admin_order_field", None)
+
+            # If attr is property check if it's fget has ordering defined
+            if isinstance(attr, property) and hasattr(attr, 'fget'):
+                admin_order_field = getattr(attr.fget, "admin_order_field")
```

Comment:
Please use single quotes.

Reply:
But line 130 uses double quotes. So things are a bit of inconsistent then.

Reply:
For new code, we're using single quotes (unless a string contains a single quote) as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).